File: JAVASCRIPT PROGRAMMING/Test61.js
"use strict";
// 61. ARMSTRONG NUMBER
const Armstrong = (num) => {
    let NUM = num;
    let res = num;
    let rem1 = 0;
    let rem2 = 0;
    let count = 0;
    let sum = 0;
    while (num !== 0) {
        rem1 = num % 10;
        count++;
        num = Math.floor(num / 10);
    }
    while (NUM != 0) {
        rem2 = NUM % 10;
        sum += Math.pow(rem2, count);
        NUM = Math.floor(NUM / 10);
    }
    if (res == sum) {
        console.log(`${sum} Is An Armstrong Number.`);
    }
    else {
        console.log(`${res} Is Not An Armstrong Number.`);
    }
};
Armstrong(153);
Armstrong(253);
Armstrong(1634);

File: PYTHON PROGRAMMING/Test61.py
# 61. PERFECT NUMBER
def Perfect(num):
    sum = 0
    for i in range(1, num):
        if num % i == 0: sum += i
    if num == sum: 
        print(f"{num} Is A Perfect Number.")
    else:
        print(f"{num} Is Not A Perfect Number.")
Perfect(28)
Perfect(27)